shrielenee.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: Shrie: a quasi summary
+---
+
+## A bit about me
+
+I'm a tech writer, avid reader, journaler, record collector, and peer-pressurer living in :sunny: Austin, TX.
+
+
+## Things I like
+
+I like some things, including but not limited to:
+
+- Flying kites in the sunshine
+- Picking through dusty record bins
```

Comment:
Check out Everyday Music across the street; awesome record collection! :musical_note: